draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,41 @@ IDNA2003 {{RFC3490}}.
+
+As a result, this document recommends limited mitigations against the privacy risks associated with cookies below. It is expected that future updates will impose stricter and more well-defined limitations on cookies, based upon experiments now being undertaken in user agents. Note that some functions currently relying on cookies might eventually be supported by separate, targeted mechanisms.
+
+Cookies are not the only mechanism that can be used to track users across sites, so while these mitigations are necessary to improve Web privacy, they are not sufficient on their own.
+
 
 ## Third-Party Cookies {#third-party-cookies}
 
-Particularly worrisome are so-called "third-party" cookies. In rendering an HTML
-document, a user agent often requests resources from other servers (such as
-advertising networks). These third-party servers can use cookies to track the
-user even if the user never visits the server directly. For example, if a user
-visits a site that contains content from a third party and then later visits
-another site that contains content from the same third party, the third party
-can track the user between the two sites.
-
-Given this risk to user privacy, some user agents restrict how third-party
-cookies behave, and those restrictions vary widly. For instance, user agents
-might block third-party cookies entirely by refusing to send Cookie header fields or
-process Set-Cookie header fields during third-party requests. They might take a less
-draconian approach by partitioning cookies based on the first-party context,
-sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another. Or they might even allow some
-third-party cookies but block others depending on user-agent cookie policy or
-user controls.
-
-This document grants user agents wide latitude to experiment with third-party
-cookie policies that balance the privacy and compatibility needs of their users.
-However, this document does not endorse any particular third-party cookie
-policy.
-
-Third-party cookie blocking policies are often ineffective at achieving their
-privacy goals if servers attempt to work around their restrictions to track
-users. In particular, two collaborating servers can often track users without
-using cookies at all by injecting identifying information into dynamic URLs.
-
-## Cookie policy {#cookie-policy}
+A cookie that is sent to a site other than the one it originated from is known as a "third-party"
+cookie. When rendering a document, a user agent can request resources from sites other than that
+where that document was obtained from (such as advertising networks). Those third-party resources
+can use cookies to track the user even if they never visit that server directly. For example, if a
```

Comment:
The key here is that third-party resources will get the same cookies no matter how they are embedded.  That's the key piece, and this doesn't quite get to that.  If the third-party server sees the same cookie in a variety of contexts, they can link those contexts to the same person (and thereby track them).

Reply:
I see how that's central to Mozilla's approach, but I don't think this level of detail / specificity is necessary here.